experimental/packages/opentelemetry-exporter-metrics-otlp-http/test/browser/CollectorMetricExporter.test.ts

```diff
@@ -240,77 +240,84 @@ describe('OTLPMetricExporter - web', () => {
         collectorExporter.export(metrics, () => {});
 
         queueMicrotask(async () => {
-          const request = server.requests[0];
-          assert.strictEqual(request.method, 'POST');
-          assert.strictEqual(request.url, 'http://foo.bar.com');
-
-          const body = request.requestBody;
-          const decoder = new TextDecoder();
-          const json = JSON.parse(
-            decoder.decode(await body.arrayBuffer())
-          ) as IExportMetricsServiceRequest;
-          // The order of the metrics is not guaranteed.
-          const counterIndex = metrics.scopeMetrics[0].metrics.findIndex(
-            it => it.descriptor.name === 'int-counter'
-          );
-          const observableIndex = metrics.scopeMetrics[0].metrics.findIndex(
-            it => it.descriptor.name === 'double-observable-gauge2'
-          );
-          const histogramIndex = metrics.scopeMetrics[0].metrics.findIndex(
-            it => it.descriptor.name === 'int-histogram'
-          );
-
-          const metric1 =
-            json.resourceMetrics[0].scopeMetrics[0].metrics[counterIndex];
-          const metric2 =
-            json.resourceMetrics[0].scopeMetrics[0].metrics[observableIndex];
-          const metric3 =
-            json.resourceMetrics[0].scopeMetrics[0].metrics[histogramIndex];
+          try {
+            const request = server.requests[0];
+            assert.strictEqual(request.method, 'POST');
+            assert.strictEqual(request.url, 'http://foo.bar.com');
+
+            const body = request.requestBody;
+            const json = JSON.parse(
+              new TextDecoder().decode(body)
+            ) as IExportMetricsServiceRequest;
```

Comment:
note: this is the only thing that really changed here, since we're not passing a `Blob` this test needed to be adapted.